system-tests/src/test/java/org/eclipse/dataspaceconnector/registration/client/RegistrationApiCommandLineClientTest.java

```diff
@@ -54,15 +55,35 @@ void listParticipants() throws Exception {
 
         assertThat(getParticipants(cmd)).noneSatisfy(p -> assertThat(p.getUrl()).isEqualTo(idsUrl));
 
-        var addCmdExitCode = cmd.execute(
-                "-c", CLIENT_DID_WEB,
+        addParticipant(cmd);
+        assertThat(getParticipants(cmd)).anySatisfy(p -> assertThat(p.getUrl()).isEqualTo(idsUrl));
```

Comment:
could we make this code robust to deprecation of `getUrl`?

Reply:
Any suggestion? Because currently only idsURl is random. But in another PR #18 we are making a change where we can use a random DID. 

So, we can leave these changes for that PR.